src/test/tests/rendering/ospray.py

```diff
@@ -45,11 +47,108 @@ def TestOsprayWindowSave():
 
     Test("ospray_window_save")
     DeleteAllPlots()
+
+    RenderingAtts = RenderingAttributes()
+    RenderingAtts.osprayRendering = 0
+    RenderingAtts.osprayShadows = 0
+    SetRenderingAttributes(RenderingAtts)
+
     CloseDatabase(silo_data_path("noise.silo"))
 
 
+def TestOsprayVolumeDomainBoundaries():
+    OpenDatabase(data_path("vtk_test_data/poiseuille_rayleigh_benard_3D_00000600.visit"))
+
+    #
+    # Change the view so that we get a good look at the domain boundaries.
+    #
+    ResetView()
+    View3DAtts = View3DAttributes()
```

Comment:
I've found that the view sometimes ends up being way off when a test is run after another test, even if the previous test doesn't explicitly change the view. In this case, you see nothing but a bank screen, which doesn't happen if you only run one test. I'm not sure why this ever happens, but I usually just run the tests manually, open up command capture, and capture whatever the view is so that I can force this view in the test. I can narrow it down and remove what isn't needed, but I'm not sure I'll have anything intelligent to say about it other than noting it's an odd behavior I sometimes see in our suite (I've also seen this in the queries tests).

Reply:
Hmmm...I think I understand that. I think what I've done in these circumstances to is to do a`ResetView()` followed by additional lines needed to change the view in the specific ways needed.